loopdk.make

```diff
@@ -211,6 +211,7 @@ projects[style_settings][subdir] = "contrib"
 projects[style_settings][download][type] = "git"
 projects[style_settings][download][url] = "http://git.drupal.org/project/style_settings.git"
 projects[style_settings][download][branch] = "7.x-2.x"
+projects[style_settings][patch][] = "https://www.drupal.org/files/issues/style_settings-fix-clobbering-of-pseudo-classes-2648698-1-7.patch"
```

Comment:
Nice catch (and patch). I suggest you add a link to https://www.drupal.org/node/2648698 to explain why this patch is needed.